app/move/views/create/person-search-results.njk

```diff
@@ -17,7 +17,7 @@
     {{ t("moves::steps::person_search_results::secondary_heading", {
       type: "person" | pluralize(resultCount),
       count: resultCount,
-      searchTerm: "123"
+      searchTerm: searchTerm
```

Comment:
Can use shorthand here

Reply:
Sadly can't as its nunjucks, not JavaScript :(